packages/shared/src/utils/table.ts

```diff
@@ -11,11 +11,11 @@ export const sortRows = (
   a: any,
   b: any,
   c: SortByDirection,
-  sortField: string,
+  sortField?: string,
   favoriteNames?: string[]
 ) => {
-  let aValue = _.get(a, sortField, '').toString();
-  let bValue = _.get(b, sortField, '').toString();
+  let aValue = _.get(a, sortField, a).toString();
+  let bValue = _.get(b, sortField, b).toString();
```

Comment:
why do we want to default to `a` and `b` ? is there any issue with existing behaviour ?
```suggestion
  let aValue = _.get(a, sortField, '').toString();
  let bValue = _.get(b, sortField, '').toString();
```

Reply:
I would prefer not to make changes in shared, unless really needed...

Reply:
it could cause a bug at other places...

Reply:
if the `a&b` is an object, to fetch the particular field from the object, we are using sortField. In my case, it is a string. so sortField is not required. 

ok! the problem is, it might break for the object which does not have sortField, I will correct this.

Reply:
```
export const sortRows = (
  a: any,
  b: any,
  c: SortByDirection,
  sortField?: string,
  favoriteNames?: string[]
) => {
  let aValue,
bValue;

  if (typeof a === 'string' && typeof b === 'string') {
    // Directly compare if both are strings
    aValue = a;
    bValue = b;
  } else {
    // Default behavior with sortField
    aValue = _.get(a, sortField, '').toString();
    bValue = _.get(b, sortField, '').toString();
  }

  if (!!favoriteNames) {
    aValue = favoriteNames.includes(aValue).toString();
    bValue = favoriteNames.includes(bValue).toString();
  }
  return sort(aValue, bValue, c);
};
```